assets/css/v2/eink/normal_header.scss

```diff
@@ -83,16 +84,10 @@
   line-height: 28px;
 }
 
-.normal-header-updated__icon {
-  display: inline-block;
-  vertical-align: top;
-}
-
 .normal-header-updated__img {
-  position: absolute;
   height: 25px;
   width: 25px;
-  margin-right: 4px;
```

Comment:
A little nitpicky, but this change put more space than [designed](https://www.figma.com/file/uyXu91sE66baBZMg81XOmk/ScreenKit-2.0?type=design&node-id=0-1544&mode=dev) between the icon and the text.